gravitee-apim-rest-api/gravitee-apim-rest-api-management-v2/gravitee-apim-rest-api-management-v2-rest/src/main/resources/openapi/management-openapi-v2.yaml

```
@@ -145,6 +145,31 @@ paths:
                     $ref: "#/components/responses/ApisResponse"
                 default:
                     $ref: "#/components/responses/Error"
+    /environments/{envId}/apis/_verify-path:
```

suggestion(non-blocking): to enable other kinds of verification, maybe we could consider having `path` as a sub-resource of the `_verify` action
```suggestion
    /environments/{envId}/apis/_verify/path:
```

---

shouldn't we also keep it plural? like `_verify/paths` ?

---

IDK if it really applies as it's a common usage to use `_doSomething` as a convention to state that an endpoint is an operation (not a resource).

---

would be more something like `/paths/_verify` if we want to rest idiomatic (pun intended)

---

I prefer @a-cordier suggestion I don't think adding (`/paths` after `_verification`) is very usual in REST APIs...

---

It's not very usual to have "action" term like `_verify` too.
And we already did such things for import and export: 
 /_import/definition
 /_export/definition

We did that because we wanted to add the possibility to export and import CRD, and maybe other format of files

Here the idea would be to allow the verification of different part of the API.
For your PR, it is the paths. But it could be the couple name+version (which must be unique in a environment), and why not, a global check on the flows configuration, or the tags.

---

For me, there are two possibilities:
1. /_verify/paths (and later /_verify/name, /_verify/flows, ...)
2. /_verify, but with a `VerifyApi` object in Input that can take more than just a list of paths.

---

OK I'll change it for `/_verify/paths` then 👌